draft-ietf-cats-framework.md

```diff
@@ -159,7 +153,15 @@ Service:
   : The same service can be provided in many locations; each of them constitutes a service instance.
 
 Computing Service:
-  : An offering is made available by a provider by orchestrating a set of computing resources (without networking resources).
+  : An offering that is made available by a provider by orchestrating a set of computing resources to process application content.
+
+  : Some differences and similarities exist between service and computing service:
+
+    * Service is packet-based, while computing service is transaction-based. Application content is exposed within a computing service.
```

Comment:
Whether this is packet based or not actually depends on the service. Let alone that we need to define what is a transaction. We don't actually need to dive into these details from a CATS perspective.

Reply:
Personally, I agree with you. Do yo mind to simplify the definition? we need to sync up with the one in use case draft. Better to send an email in the CATS ML